builder.go

```diff
@@ -83,48 +73,42 @@ func replaceEnv(cmds []string) {
 	}
 }
 
-func (b *Bob) Close() error {
+func (b *Bob) Close() {
 	close(b.done)
-	return b.w.Close()
 }
 
 func (b *Bob) Watch(path string) error {
 	b.watchDir = path
-	// this can never return false since we will always
-	// have at least one file in the directory (.snag.yml)
-	_ = b.watch(path)
+
 	b.execute()
 
+	// Make the channel buffered to ensure no event is dropped. Notify will drop
+	// an event if the receiver is not able to keep up the sending pace.
+	c := make(chan notify.EventInfo, 1)
```

Comment:
Events could still technically be dropped with a buffered channel if queueing up the build takes too long. If we drop a couple of events is that an issue?